client/src/services/utilities.js

```diff
@@ -31,7 +31,7 @@ type ShowNumberCurrencyProps = {
 }
 
 export const ShowNumberCurrency = ({num, cur = '€'}: ShowNumberCurrencyProps) => {
-  if (!num) return null
+  if (num == null || isNaN(num)) return null
```

Comment:
Flow declares `number` as input, so I'm not checking other falsy values such as `false` or `''`

Reply:
It never ceases to amaze me, that `isNaN(null)` is `false` :roll_eyes:

Reply:
But maybe you could use lodash's `isFinite`. It's up to you.